Doc/library/logging.rst

```diff
@@ -1111,6 +1111,14 @@ functions.
    .. note:: If you are thinking of defining your own levels, please see the
       section on :ref:`custom-levels`.
 
+.. function:: getLevelNamesMapping()
+
+   Returns a mapping from level names to their corresponding logging
```

Comment:
Maybe add an example, like 'For example, the string "FATAL" maps to `logging.FATAL`.' Or maybe it's clear enough without this. @vsajip : what are your thoughts?

Reply:
> Maybe add an example, like 'For example, the string "FATAL" maps to `logging.FATAL`.' Or maybe it's clear enough without this. @vsajip : what are your thoughts?

I agree that an example would be good.

Reply:
I've used CRITICAL for the example as it's used in a few other places in the doc (and is an alias for FATAL).